app.rb

```diff
@@ -1,9 +1,25 @@
 require 'sinatra/base'
+require './lib/peeps'
+require 'sinatra/reloader'
 
 class Chitter < Sinatra::Base
-  get '/test' do
-    'Test page'
+  configure :development do
+    register Sinatra::Reloader 
   end
+  get '/' do
```

Comment:
To add to this - this is particularly great because there's a clear separation between the Model, Views and Controller, making this a nice demonstration of an MVC project.